tests/_support/loadGrunt.ts

```diff
@@ -0,0 +1,79 @@
+import * as grunt from 'grunt';
+import * as path from 'path';
+import * as mockery from 'mockery';
+import * as _ from 'lodash';
+import { IRootRequire } from 'dojo/loader';
+
+declare const require: IRootRequire;
+
+export interface MockList {
+	[key: string]: any;
+}
+
+export interface TaskLoadingOptions {
+	peerDependencies?: {
+		[key: string]: any;
+	};
+}
+
+export function runGruntTask(taskName: string, callback?: () => void) {
+	const task = (<any> grunt.task)._taskPlusArgs(taskName);
+
+	return task.task.fn.apply({
+		nameArgs: task.nameArgs,
+		name: task.task.name,
+		args: task.args,
+		flags: task.flags,
+		async: function () {
+			return callback;
+		}
+	}, task.args);
+}
+
+function registerMockList(mocks: MockList) {
+	const keys = Object.keys(mocks);
+
+	for (let i = 0; i < keys.length; i++) {
+		mockery.registerMock(keys[i], mocks[keys[i]]);
+	}
+}
+
+export function loadTasks(mocks?: MockList, options?: TaskLoadingOptions) {
+	mockery.enable({
+		warnOnReplace: false,
+		warnOnUnregistered: false,
+		useCleanCache: true
+	});
+	mockery.resetCache();
+
+	mockery.registerMock('lodash', _);
```

Comment:
we're not using lodash (although grunt does). I'm guessing `grunt-dojo2` uses and mocks it like this so they can mock a known lodash if/when the need arises.